src/mongoc/mongoc-bulk-operation.h

```diff
@@ -31,6 +31,8 @@
 
 BSON_BEGIN_DECLS
 
+/* forward decl */
+typedef struct _mongoc_client_session_t mongoc_client_session_t;
```

Comment:
@ajdavis: Your example in https://github.com/mongodb/mongo-c-driver/blob/0dc8fb1aca4194df67afd431c55d438f9ecef56c/src/mongoc/mongoc-apm-private.h#L30-L29 was in a private header and I don't think we want `_mongoc_client_session_t` in the public ABI, so I didn't adapt it verbatim.

Is this kosher?

Reply:
Changed to `struct _mongoc_client_session_t`.

Reply:
I'm not seeing that change in this GitHub PR.

Reply:
Looks like I updated the function but forgot to change this forward decl. Will revise that Monday morning.